src/core/dpl-cms/dpl-cms.ts

```diff
@@ -29,7 +35,7 @@ type AwaitedInput<T> = PromiseLike<T> | T;
 type Awaited<O> = O extends AwaitedInput<infer T> ? T : never;
 
 /**
- * @summary Get campaign matching search result facets
+ * @summary Hent kampagne som matcher facetter fra søgeresultatet
```

Comment:
Tænker dette burde forblive engelsk

Reply:
It is auto generated code that comes from https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/890/files